src/main/scripts/src/app/core/rest/organization.service.ts

```diff
@@ -39,15 +38,15 @@ export class OrganizationService {
     return this.httpClient.get<Organization>(OrganizationService.apiPrefix(code));
   }
 
-  public deleteOrganization(code: string): Observable<Response> {
+  public deleteOrganization(code: string): Observable<Object> {
```

Comment:
This is the default return type of several `HttpClient` methods. However, I used it just temporarily in order to fix type errors. We can discuss what is the best return type for our service methods.